universal-application-tool-0.0.1/app/services/program/BlockDefinition.java

```diff
@@ -0,0 +1,48 @@
+package services.program;
+
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+import java.util.Optional;
+
+/** Defines a single program block, which contains a list of questions and data about the block. */
+@AutoValue
+public abstract class BlockDefinition {
+
+  public abstract String name();
+
+  public abstract String description();
```

Comment:
Suggestion (non-blocking): recommend adding docstrings for the members of these value classes ([see example](https://github.com/seattle-uat/universal-application-tool/pull/115/files#diff-9754168af8d327c9a874a1144eb229af2041a9112a50b0c5f0232a4d3ffcf613)). That will go a long ways toward making the code friendlier to city staff and open source contributors.

Reply:
done!